src/main.spec.js

```diff
@@ -36,21 +36,66 @@ test("Checks HTMLElements", () => {
       name: "FinalDir",
       absolute_path: "/mnt/project/FinalDir",
       type: "directory",
-      children: null
+      children: [
+        {
+          name: "yarn.lock",
+          absolute_path: "/mnt/project/vendor/yarn.lock",
+          type: "file",
+          children: null
+        },
+        {
+          name: "package.json",
+          absolute_path: "/mnt/project/vendor/package.json",
+          type: "file",
+          children: null
+        }
+      ]
     }
   ];
   const container = document.createElement("div");
   fsapi.render(input, container);
 
-  const ul = container.querySelector("ul");
+  const trees = container.querySelectorAll("ul");
 
-  for (const child of ul.children) {
+  for (const child of trees[0].children) {
     expect(child).toBeInstanceOf(HTMLLIElement);
   }
-  expect(ul.children[0].textContent).toBe("Dir");
-  expect(ul.children[1].textContent).toBe("FinalDir");
-  expect(ul.children[2].textContent).toBe("AnotherFile.html");
-  expect(ul.children[3].textContent).toBe("SFile.sth");
-  expect(ul.children[4].textContent).toBe("test.go");
-  expect(ul.children[5].textContent).toBe("Troll.go");
+  expect(trees[0].children[0].children[0].textContent).toBe("Dir");
+  expect(trees[0].children[1].children[0].textContent).toBe("FinalDir");
+  expect(trees[0].children[2].children[0].textContent).toBe("AnotherFile.html");
+  expect(trees[0].children[3].children[0].textContent).toBe("SFile.sth");
+  expect(trees[0].children[4].children[0].textContent).toBe("test.go");
+  expect(trees[0].children[5].children[0].textContent).toBe("Troll.go");
+
+  expect(
+    trees[0].children[1].children[1].children[0].children[0].textContent
```

Comment:
Hmmm... I think this is a bit illegible here. What about replacing `trees[0].children[1].children[1]` with something more semantic and reusing it also in the test below?

e.g.

```
const vendorChildren = trees[0].children[1].children[1];  // or vendorSubTree, or something else that sounds more relevant
```